virttest/utils_time.py

```diff
@@ -212,3 +212,32 @@ def update_clksrc(vm, clksrc=None):
     if clksrc and clksrc != 'kvm-clock':
         boot_option_added = "clocksource=%s" % clksrc
         utils_test.update_boot_option(vm, args_added=boot_option_added)
+
+
+def is_ntp_enabled(session):
+    """
+    Get current NTP state for guest/host
+    """
+    cmd = 'timedatectl |grep "NTP enabled"'
+    output = execute(cmd, session=session)
+    if 'yes' in output.split(":")[1].strip():
+        return True
+    else:
+        return False
+
+
+def ntp_switch(session, off=True):
+    """
+    Turn off/on ntp for guest/host
+    """
+    cmd = "timedatectl set-ntp"
+    if off:
+        cmd += " 0"
+    else:
+        cmd += " 1"
+    execute(cmd, session=session)
```

Comment:
let's get the return of execute() and log when we raise TestError in L241 and L243